app/aws/metrics.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations  # PEP 563 -- Postponed Evaluation of Annotations
+
+from typing import TYPE_CHECKING
+
+from aws_embedded_metrics import metric_scope  # type: ignore
+from aws_embedded_metrics.config import get_config  # type: ignore
+from botocore.exceptions import ClientError
+from flask import current_app
+
+from app.config import Config
+
+if TYPE_CHECKING:  # A special Python 3 constant that is assumed to be True by 3rd party static type checkers
+    from aws_embedded_metrics.logger.metrics_logger import MetricsLogger  # type: ignore
+
+    from app.queue import RedisQueue
+
+Metrics_Config = get_config()
```

Comment:
To follow usual variable and lower snake case conventions:

```suggestion
metrics_config = get_config()
```